xapian-core/tests/api_sorting.cc

```diff
@@ -321,79 +321,77 @@ DEFINE_TESTCASE(sortfunctorremote1, remote) {
 }
 
 DEFINE_TESTCASE(replace_weights, backend) {
-    // open the database (in this case a simple text file
-    // we prepared earlier)
 
```

Comment:
You can get rid of these blank lines, too. Try to make it look as much like `sortfunctortemote1` above as possible.